builds/delete-serverless-service-build-pack/Jenkinsfile

```diff
@@ -1232,3 +1232,26 @@ def parseJsonMap(jsonString) {
 	lazyMap.putAll(jsonObj)
 	return lazyMap
 }
+
+def removePermissionAndDeleteRepo(serviceId) {
```

Comment:
Please move this code to a separate module.

Reply:
Moved to scm-module file